src/App.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+//Create an empty array and store it in a var named todoList
+let todoList = [
+  { "id": 1, 
+  "title": "Workout"
+  },
+  { "id": 2, 
+  "title": "Work"
+  },
+  { "id": 3, 
+  "title": "Work on homework"
+  }
+];
+
+function App() {
+  return (
+    <div>
+      <header>
```

Comment:
@Karla1981 You would not want to use a header tag here, this is more reserved for navigation or a summary block on a webpage.  Putting the content inside the div will work fine.